app/code/Magento/InventoryBundleIndexer/Test/_files/source_items_bundle.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
What is the main reason to have this fixture?
We have same data in `C:\Users\vnayda\PhpstormProjects\engcom_public\app\code\Magento\InventoryApi\Test\_files\source_items.php`

Reply:
Fixed.